bin/post-checkout.js

```diff
@@ -0,0 +1,10 @@
+// @noflow
+
+const [origHead, head, flag] = process.env.HUSKY_GIT_PARAMS.split(' ');
```

Comment:
I only removed the "compatiblity layer" for the older husky, because we don't have it here.